tutorials/convert_lit_models.md

```diff
@@ -1,38 +1,39 @@
+Several models retrieved from external sources must be reformatted with naming conventions for Lit-GPT weights before finetuning. We've provided a helpful script to convert models finetuned with certain Lit-GPT techniques back to their original format.
 
 The way the script is used depends on the finetuning method. The commands for converting a model back to its original format after using one of the several finetuning methods supported by Lit-GPT are shown below.
 
 > [!NOTE]\
-> each example shown below uses Falcon-7B please be sure to update --checkpoint_dir accordingly
+> all examples use the Meta Llama-2-7b-chat-hf checkpoints and Alpaca dataset during finetuning
 
-### Full Finetuning
-
-After full finetuning, your checkpoint directory will contain a file named `lit_model_finetuned.pth`and converting the finetuned model back to its original weights naming convention can be done by setting the `--checkpoint_name` with:
+> [!NOTE]\
+> converting Adapter and Adapter V2 finetuned checkpoints cannot be supported until there is a HF model with adapter weights
 
-```sh
-python scripts/convert_lit_checkpoint.py \
-    --checkpoint_name lit_model_finetuned.pth \
-    --checkpoint_dir checkpoints/tiiuae/falcon-7b
-```
+> [!NOTE]\
+> finetuned checkpoints will be saved to a subdirectory in a main directory named `out` e.g. finetuning with LoRA will save checkpoints in `out/lora/alpaca`. that new subdirectory is used as the `--out_dir` argument provided to convert_lit_checkpoint
 
-### Adapter and Adapter V2 Finetuning
+### Full Finetuning
 
-After finetuning with either Adapter technique, your checkpoint directory will contain a file named `lit_model_adapter_finetuned.pth` and converting the finetuned model back to its original weights naming convention can be done by setting the `--checkpoint_name` with:
+After full finetuning, your `out_dir` will contain a file named `lit_model_finetuned.pth` and converting the finetuned model back to its original weights naming convention can be done with the following:
 
 ```sh
 python scripts/convert_lit_checkpoint.py \
-    --checkpoint_name lit_model_adapter_finetuned.pth \
-    --checkpoint_dir checkpoints/tiiuae/falcon-7b
+    --checkpoint_name lit_model_finetuned.pth  \
+    --out_dir out/full/alpaca/ \
+    --model_name Llama-2-7b-chat-hf
 ```
 
 ### LoRA Finetuning
 
-After finetuning with LoRA, your checkpoint directory will contain a file named `lit_model_lora_finetuned.pth` and converting the finetuned model back to its original weights naming convention can be done by setting the `--checkpoint_name` with:
+After finetuning with LoRA, your `out_dir` will contain a file named `lit_model_lora_finetuned.pth` and converting the finetuned model back to its original weights naming convention can be done with the following:
+
+> [!NOTE]\
+> before conversion, merge LoRA weights with `lit_gpt.lora.merge_lora_weights(your_model)``
```

Comment:
Are you cool with me adding this into convert_lit_checkpoint? This seems like something I should enable in the script, instead of leaving it to the users.

Reply:
if yes, I will rename this PR accordingly.

Reply:
implemented in [5d87fa9](https://github.com/Lightning-AI/lit-gpt/pull/362/commits/5d87fa924e8ac4e2932855bfb6e9c8eedf91e65f)